src/OpenDB/src/swig/common/swig_common.cpp

```diff
@@ -290,4 +290,12 @@ void createSBoxes(odb::dbSWire* swire,
   for (odb::Point point : points)
     odb::dbSBox::create(
         swire, via, point.getX(), point.getY(), type);
+}
+
+odb::dbDatabase* create_db()
+{
+  utl::Logger* logger = new utl::Logger();
```

Comment:
@maliberty This swig_common.cpp is part of the SWIG'd source code how should the logger for openROAD be accessed in this context?

Reply:
The swig wrappers are the rare place where you should access the openroad singleton and you can get the logger from there.